hpx/runtime.hpp

```diff
@@ -314,22 +412,50 @@ namespace hpx
         notification_policy_type::on_error_type on_error_func(
             notification_policy_type::on_error_type&&);
 
+        virtual std::uint32_t get_locality_id(error_code& ec)
+        {
+            return 0;
+        }
+
+        virtual std::size_t get_num_worker_threads()
```

Comment:
Same here (and below): could this be `const`?

Reply:
I'm pretty sure it can. Thanks!